### ISSUE TITLE
[Exception][JetBrainsClient 2023.2] com.intellij.diagnostic.PluginException: ai.codium.ui.actions.CreateCommitMessage...

### ISSUE THREAD

**Issue by plerbach-mfl** [Codium-ai/codiumai-jetbrains-release#133]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
com.intellij.diagnostic.PluginException: ai.codium.ui.actions.CreateCommitMessageAction (CodiumAI.Commit.Button): group with id "Vcs.MessageActionGroup" isn't registered; action will be added to the "Other" group (module=PluginDescriptor(name=CodiumAI - Integrity Agent powered by GPT-3.5&4, id=codiumai.codiumai, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\JetBrainsClient232.8660.185\plugins\codium-ai-plugin-0.6.14-all.jar, version=0.6.14, package=null, isBundled=false))
	at com.intellij.openapi.actionSystem.impl.ActionManagerImplKt.reportActionError(ActionManagerImpl.kt:1568)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImplKt.access$reportActionError(ActionManagerImpl.kt:1)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getParentGroup(ActionManagerImpl.kt:763)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.processAddToGroupNode(ActionManagerImpl.kt:702)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.processActionElement(ActionManagerImpl.kt:426)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerPluginActions(ActionManagerImpl.kt:272)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerActions$intellij_platform_ide_impl(ActionManagerImpl.kt:145)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.<init>(ActionManagerImpl.kt:118)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:63)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:1011)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:47)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:43)
	at com.intellij.serviceContainer.BaseComponentAdapter.doCreateInstance(BaseComponentAdapter.kt:158)
	at com.intellij.serviceContainer.BaseComponentAdapter.createInstance(BaseComponentAdapter.kt:137)
	at com.intellij.serviceContainer.BaseComponentAdapter.access$createInstance(BaseComponentAdapter.kt:21)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invokeSuspend(BaseComponentAdapter.kt:185)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceAsync(BaseComponentAdapter.kt:180)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsyncIfDefined(ComponentManagerImpl.kt:646)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsync(ComponentManagerImpl.kt:637)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invokeSuspend(appServicePreloading.kt:179)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.diagnostic.TracerKt.subtask(tracer.kt:45)
	at com.intellij.diagnostic.TracerKt.subtask$default(tracer.kt:34)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2.invokeSuspend(appServicePreloading.kt:105)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)

```

# Other Information
```
=== About ===
Build version: JetBrains Client 2023.2 Build: #JBC-232.8660.185 July 26, 2023
Theme: Dark
JRE: 17.0.7+7-b1000.6, JetBrains s.r.o.
JVM: 17.0.7+7-b1000.6, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 10.0 (amd64)

=== Plugins ===
Custom plugins: [CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.14), GitHub Copilot (1.3.2.3479), google-java-format (1.13.0.0)]
Disabled plugins:[]
```

**Comment by aldanchenko** [comment]
Hi @plerbach-mfl 

Thanks for reporting! We will check it and fix.

Best regards,
Alex

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.15
Thank you again for reporting!
